write/csvTable.go

```diff
@@ -1,420 +0,0 @@
-package write
```

Comment:
Can you help me understand why these `write/*` deletions are here?

Reply:
The conversion of CSV to line protocol was refactored and improved in https://github.com/influxdata/influxdb/tree/master/pkg/csv2lp  as requested by https://github.com/influxdata/influxdb/pull/17599#issuecomment-614174079